redirects-transition.conf

```diff
@@ -1336,6 +1342,8 @@ rewrite ^/pubrec/cfsdd/cfsdd.shtml https://www.fec.gov/introduction-campaign-fin
 rewrite ^/pubrec/cfsdd/cfsded_000.pdf https://www.fec.gov/resources/cms-content/documents/cfsded.pdf redirect;
 
 # rad/ redirects
+rewrite ^/rad/analyst_search.shtml https://www.fec.gov/help-candidates-and-committees/question-rad/ redirect;
+rewrite ^/rad/best_practices_to_avoid_pitfalls_2018.pdf https://www.fec.gov/help-candidates-and-committees/filing-reports/#troubleshooting redirect;
```

Comment:
Redirects to anchor links don't work. Need to take off the #troubleshooting part.